dart/lib/src/utils.dart

```diff
@@ -20,3 +20,7 @@ String formatDateAsIso8601WithMillisPrecision(DateTime date) {
 
 /// helper to detect a browser context
 const isWeb = identical(0, 0.0);
+
+/// helper to detect if app is in release mode
+const isReleaseMode =
+    bool.fromEnvironment('dart.vm.product', defaultValue: false);
```

Comment:
is the other one flutter only?

Reply:
yes, it's a constant from Flutter foundation